tests/benchmarks/cpu/benchmark_propagator.cpp

```diff
@@ -76,20 +76,47 @@ auto toy_cfg =
     toy_det_config{}.n_brl_layers(4u).n_edc_layers(7u).do_check(false);
 
 void fill_tracks(vecmem::vector<free_track_parameters<algebra_t>> &tracks,
-                 const std::size_t theta_steps, const std::size_t phi_steps) {
-    // Set momentum of tracks
-    const scalar mom_mag{10.f * unit<scalar>::GeV};
+                 const std::size_t n_tracks, bool do_sort = true) {
+    using scalar_t = dscalar<algebra_t>;
+    using uniform_gen_t =
+        detail::random_numbers<scalar_t,
+                               std::uniform_real_distribution<scalar_t>>;
+    using trk_generator_t =
+        random_track_generator<free_track_parameters<algebra_t>, uniform_gen_t>;
+
+    trk_generator_t::configuration trk_gen_cfg{};
+    trk_gen_cfg.seed(42u);
+    trk_gen_cfg.n_tracks(n_tracks);
+    trk_gen_cfg.phi_range(-constant<scalar_t>::pi, constant<scalar_t>::pi);
+    trk_gen_cfg.eta_range(-4.f, 4.f);
```

Comment:
toy detector can have tracks at eta of 4?

Reply:
Changed to |eta| = 3